src/mainwindow.cpp

```diff
@@ -1895,6 +1924,11 @@ void MainWindow::setTheme(Theme theme)
     m_currentTheme = theme;
     switch (theme) {
     case Theme::Light: {
+        QJsonObject themeData {
```

Comment:
I don't like this way of changing themes, but I'll add this to my CSS themes PR when this is merged (or if my PR is merged first, I'll look into updating it here as well)

Reply:
What do you suggest?
Should I just send the theme type and in the .qml files read the appropriate properties from your .css files? (if it's possible)

Reply:
This is merely to signal to the .qml files that the theme changed with the data (type, colors...).

Reply:
I  don't really know much about qml, but if that's possible that would be great. I think we should try putting all CSS in the same place.

Reply:
Maybe it's enough to just change the values I set in the lines you highlighted to use your .css file values and the .qml will just pick it up from there (therefore, no hard-coded styles in .qml)